<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/back.jpg);
background-repeat: repeat;
border: 1px solid #E0E0E0;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>400</height>
         </size>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>276</width>
           <height>400</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignmentFlag::AlignLeft">
           <widget class="QPushButton" name="freePenButton">
            <property name="minimumSize">
             <size>
              <width>102</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/button2.png);
background-repeat: no-repeat;
background-position: center;
min-width: 100px;
min-height: 30px;font-weight: bold;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>画笔</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="circleButton">
            <property name="minimumSize">
             <size>
              <width>102</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/button2.png);
background-repeat: no-repeat;
background-position: center;
min-width: 100px;
min-height: 30px;font-weight: bold;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>圆形</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lineButton">
            <property name="minimumSize">
             <size>
              <width>102</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/button2.png);
background-repeat: no-repeat;
background-position: center;
min-width: 100px;
min-height: 30px;font-weight: bold;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>直线</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rectButton">
            <property name="minimumSize">
             <size>
              <width>102</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/button2.png);
background-repeat: no-repeat;
background-position: center;
min-width: 100px;
min-height: 30px;font-weight: bold;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>矩形</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>40</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>颜色:</string>
             </property>
            </widget>
            <widget class="QPushButton" name="colorButton">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>0</y>
               <width>60</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>颜色</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <widget class="QLabel" name="label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>40</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>线宽:</string>
             </property>
            </widget>
            <widget class="QSpinBox" name="penWidthSpinBox">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>0</y>
               <width>60</width>
               <height>30</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="undoButton">
            <property name="minimumSize">
             <size>
              <width>102</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/button2.png);
background-repeat: no-repeat;
background-position: center;
min-width: 100px;
min-height: 30px;
font-weight: bold;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>撤销</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>102</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(C:/aaa_learn/QT_draw/Draw/resources/button2.png);
background-repeat: no-repeat;
background-position: center;
min-width: 100px;
min-height: 30px;
font-weight: bold;
font-size: 12px;</string>
            </property>
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
